src/Microsoft.Extensions.DependencyInjection.Abstractions/EnumerableServiceDescriptor.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public class EnumerableServiceDescriptor: ServiceDescriptor
```

Comment:
Yes you have to downcast. We're willing a limited set of APIs for 1.1. This is one of them.